azure-group-deploy.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-while getopts "a:l:g:s:f:e:u" opt; do
+while getopts "a:l:g:s:f:e:u:v" opt; do
```

Comment:
'u' should not have a colon after it, since it does not require an option argument